x-pack/plugin/eql/qa/common/src/main/resources/mapping-default.json

```diff
@@ -46,8 +46,7 @@
             "type" : "date"
         },
         "@timestamp" : {
-            "type" : "alias",
-            "path" : "timestamp"
+            "type" : "date"
```

Comment:
Because timestamp is not unix time but rather window filetime  (see [this comment](https://github.com/elastic/elasticsearch/pull/58635#discussion_r446565395) and until the dataset gets updated, working on a separate field is the cleaner.